packages/evidence/scripts/build-template.js

```diff
@@ -116,7 +102,8 @@ fsExtra.outputFileSync(
 				}
 			}
 		},
-		customLogger: logger
+		customLogger: logger,
+		logLevel: 'silent'
```

Comment:
Adding `logLevel: 'silent'` seems to remove all the SvelteKit output info from the logs. Is that intentional? I feel like the SvelteKit info could be helpful in debugging.

<details>
<summary>Without <code>logLevel: 'silent'</code></summary>

```
❯ pnpm build --debug

> my-evidence-project@0.0.1 build /home/zach/code/evidence/evidence/e2e/basic
> evidence build "--debug"

Evidence running with debug logging
[DEBUG]:  vite v5.4.6 building SSR bundle for production...
[DEBUG]:  ✓ 2078 modules transformed.
No sources found, execute "npm run sources" to generate
[DEBUG]:  vite v5.4.6 building for production...
[DEBUG]:  [plugin:vite:resolve] [plugin vite:resolve] Module "fs/promises" has been externalized for browser compatibility, imported by "/home/zach/code/evidence/evidence/e2e/basic/.evidence/template/src/pages/+layout.js". See https://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility for more details.
[DEBUG]:  ✓ 2866 modules transformed.
[DEBUG]:  [plugin:vite:reporter] [plugin vite:reporter]
(!) /home/zach/code/evidence/evidence/node_modules/.pnpm/prismjs@1.29.0/node_modules/prismjs/prism.js is dynamically imported by /home/zach/code/evidence/evidence/packages/ui/core-components/dist/unsorted/ui/prismLoader.js but also statically imported by /home/zach/code/evidence/evidence/packages/ui/core-components/dist/unsorted/ui/QueryViewerSupport/Prismjs.svelte, /home/zach/code/evidence/evidence/packages/ui/core-components/dist/unsorted/ui/prism-svelte.js, dynamic import will not move module into another chunk.

[DEBUG]:  .svelte-kit/output/client/_app/version.json                                                                 0.03 kB │ gzip:     0.05 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/wordmark-gray-800.CnzB_64T.png                             11.87 kB
[DEBUG]:  .svelte-kit/output/client/.vite/manifest.json                                                              32.67 kB │ gzip:     3.14 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-ExtraLight.HPQmpGpQ.woff2                         55.57 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-ExtraLightItalic.xW7ezc0a.woff2                   58.00 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Regular.BE1HJwUh.woff2                            58.09 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Italic.wSPWDyg7.woff2                             58.78 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Light.TIhg4DnI.woff2                              58.91 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-ExtraBold.DeRaT2m-.woff2                          59.18 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-ExtraBoldItalic.MQ9B1Gco.woff2                    59.74 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-LightItalic.zW0f9iJS.woff2                        60.67 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-SemiBold.zu1TM2FZ.woff2                           60.88 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Medium.NsGaEaui.woff2                             61.13 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Bold.BwJz_A1v.woff2                               61.29 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-MediumItalic.BvSK_s7Y.woff2                       61.88 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-SemiBoldItalic.D3fgUf4W.woff2                     62.13 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-BoldItalic.CQC_WCFZ.woff2                         62.16 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-ExtraLight.HAWHIZWB.woff                          69.52 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Regular.C0aqGnDS.woff                             72.05 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-ExtraLightItalic.C9nFbxmj.woff                    72.68 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Light.DBct1VnS.woff                               72.94 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-ExtraBold.Dm6k0JyR.woff                           73.01 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Italic.3s0_PsVL.woff                              73.28 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-ExtraBoldItalic.SZr5i_vZ.woff                     74.36 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-SemiBold.CCUIByHq.woff                            75.12 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-LightItalic.B3o_X74d.woff                         75.20 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Medium.BZl4fq6u.woff                              75.20 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-Bold.C57huwyI.woff                                75.81 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-MediumItalic.De40Wj9n.woff                        76.62 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-SemiBoldItalic.DjbDqXej.woff                      76.77 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Spectral-BoldItalic.CQquRzSI.woff                          76.91 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Regular.CKDp9E3C.woff2                               98.87 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Thin.CSJ_owlW.woff2                                  99.63 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Black.DgUKMdlB.woff2                                102.87 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ExtraLight.egkqWonf.woff2                           104.23 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Light.CdXm_ABL.woff2                                104.33 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-SemiBold.Ctx7G98q.woff2                             105.80 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Medium.P1cOs5ep.woff2                               105.92 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ExtraBold.B0QOs-T2.woff2                            106.11 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Bold.CuhepTt8.woff2                                 106.14 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ThinItalic.B6FKq6qK.woff2                           106.50 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Italic._3PMmu0i.woff2                               106.88 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-BlackItalic.ZFcDzO3e.woff2                          108.75 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-LightItalic.COmGGTuG.woff2                          111.33 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ExtraLightItalic.Bf25KIHs.woff2                     111.39 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ExtraBoldItalic.OErwaGem.woff2                      111.71 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-BoldItalic.R15IsAnq.woff2                           111.81 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-SemiBoldItalic.BNz1Al2H.woff2                       112.05 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-MediumItalic.DWiIAHvp.woff2                         112.18 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Regular.DJOZHnwz.woff                               133.84 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Thin.CVMZtQER.woff                                  135.92 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Black.DtxVgCT8.woff                                 138.76 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Light.CG9upfZ4.woff                                 140.63 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ExtraLight.CbFdmbHh.woff                            140.72 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Medium.oHczPpWt.woff                                142.55 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ExtraBold.B8imQN9B.woff                             142.92 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-SemiBold.BHlX_6qk.woff                              142.93 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Bold.2PT7Q-hu.woff                                  143.21 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-Italic.Ci_5KQU1.woff                                144.37 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ThinItalic.B2bFwwDp.woff                            145.48 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-BlackItalic.CkZ-ktDH.woff                           146.82 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ExtraLightItalic.B3gY-m5W.woff                      150.00 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-LightItalic.C2ZuFZOC.woff                           150.09 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-ExtraBoldItalic.Bt8eprh6.woff                       150.63 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-MediumItalic.8laNK8GF.woff                          150.99 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-BoldItalic.C906RMVC.woff                            151.05 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/Inter-SemiBoldItalic.CxeAX7g8.woff                        151.18 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/workers/duckdb-browser-eh.worker-CKjxBIML.js                     264.42 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/workers/duckdb-browser-mvp.worker-C_5zlvj5.js                    317.62 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/duckdb-eh.Bbw6HAmJ.wasm                                18,105.92 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/duckdb-mvp.IKz61icS.wasm                               22,092.38 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/VennDiagram.O0wiVdv0.css                                   84.01 kB │ gzip:    18.64 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/assets/0.BvXVVLdn.css                                             86.10 kB │ gzip:    14.83 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/entry/start.CUL3zLeI.js                                            0.06 kB │ gzip:     0.08 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/duckdb-eh.4_7A-zoJ.js                                       0.10 kB │ gzip:     0.11 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/duckdb-mvp.DtaEw-8Q.js                                      0.10 kB │ gzip:     0.11 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/index.CBzv_x0-.js                                           0.12 kB │ gzip:     0.11 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/duckdb-browser-eh.worker.DipH1UWk.js                        0.26 kB │ gzip:     0.18 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/duckdb-browser-mvp.worker.CnvUlyNd.js                       0.26 kB │ gzip:     0.17 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/nodes/2.BQv03tzD.js                                                0.36 kB │ gzip:     0.26 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/stores.CkqeuMbJ.js                                          0.66 kB │ gzip:     0.27 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/profile.Cdpsgq_9.js                                         1.88 kB │ gzip:     0.73 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/preload-helper.BelkbqnE.js                                  1.95 kB │ gzip:     0.83 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/nodes/3.DTt1kStJ.js                                                2.02 kB │ gzip:     0.83 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/prism-python.bY0kyGTd.js                                    2.51 kB │ gzip:     1.21 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/prism-sql.BmO7cOkP.js                                       3.45 kB │ gzip:     1.94 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/prism-svelte.rokHYIJk.js                                    4.13 kB │ gzip:     1.19 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/prism-bash.Bi6OIcdJ.js                                      9.15 kB │ gzip:     4.00 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/Button.DCLs5t-a.js                                          9.62 kB │ gzip:     2.50 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/nodes/4.ChYHIdtT.js                                               10.60 kB │ gzip:     2.27 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/prism-markdown.Dc3yZWyV.js                                 10.68 kB │ gzip:     3.52 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/nodes/5.D-8NUYzu.js                                               16.48 kB │ gzip:     4.10 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/nodes/1.NaKHOTtT.js                                               19.36 kB │ gzip:     4.05 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/index.OljE50Gs.js                                          19.87 kB │ gzip:     5.96 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/nodes/6.DJiqtUYd.js                                               22.24 kB │ gzip:     4.77 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/entry/app.wHupYe-E.js                                             25.31 kB │ gzip:     3.96 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/scheduler.B8ZnmH6g.js                                      37.60 kB │ gzip:    10.92 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/entry.CNAphrWy.js                                          72.65 kB │ gzip:    19.88 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/AccordionItem.BFat6zXO.js                                  92.98 kB │ gzip:    11.15 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/nodes/7.D7AZl8an.js                                              429.68 kB │ gzip:    73.88 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/inferColumnTypes.BL667tx9.js                              604.89 kB │ gzip:   118.72 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/nodes/0.Cxmnz08-.js                                              781.74 kB │ gzip:   153.22 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/index.BChc4V48.js                                       5,322.59 kB │ gzip: 2,021.36 kB
[DEBUG]:  .svelte-kit/output/client/_app/immutable/chunks/VennDiagram.svelte_svelte_type_style_lang.qFAo7pv-.js   5,631.76 kB │ gzip: 1,234.19 kB
[DEBUG]:  ✓ built in 24.34s
No sources found, execute "npm run sources" to generate
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/wordmark-gray-800.CnzB_64T.png                                 11.87 kB
[DEBUG]:  .svelte-kit/output/server/.vite/manifest.json                                                                  31.80 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-ExtraLight.HPQmpGpQ.woff2                             55.57 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-ExtraLightItalic.xW7ezc0a.woff2                       58.00 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Regular.BE1HJwUh.woff2                                58.09 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Italic.wSPWDyg7.woff2                                 58.78 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Light.TIhg4DnI.woff2                                  58.91 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-ExtraBold.DeRaT2m-.woff2                              59.18 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-ExtraBoldItalic.MQ9B1Gco.woff2                        59.74 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-LightItalic.zW0f9iJS.woff2                            60.67 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-SemiBold.zu1TM2FZ.woff2                               60.88 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Medium.NsGaEaui.woff2                                 61.13 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Bold.BwJz_A1v.woff2                                   61.29 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-MediumItalic.BvSK_s7Y.woff2                           61.88 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-SemiBoldItalic.D3fgUf4W.woff2                         62.13 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-BoldItalic.CQC_WCFZ.woff2                             62.16 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-ExtraLight.HAWHIZWB.woff                              69.52 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Regular.C0aqGnDS.woff                                 72.05 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-ExtraLightItalic.C9nFbxmj.woff                        72.68 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Light.DBct1VnS.woff                                   72.94 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-ExtraBold.Dm6k0JyR.woff                               73.01 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Italic.3s0_PsVL.woff                                  73.28 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-ExtraBoldItalic.SZr5i_vZ.woff                         74.36 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-SemiBold.CCUIByHq.woff                                75.12 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-LightItalic.B3o_X74d.woff                             75.20 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Medium.BZl4fq6u.woff                                  75.20 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-Bold.C57huwyI.woff                                    75.81 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-MediumItalic.De40Wj9n.woff                            76.62 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-SemiBoldItalic.DjbDqXej.woff                          76.77 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Spectral-BoldItalic.CQquRzSI.woff                              76.91 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Regular.CKDp9E3C.woff2                                   98.87 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Thin.CSJ_owlW.woff2                                      99.63 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Black.DgUKMdlB.woff2                                    102.87 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ExtraLight.egkqWonf.woff2                               104.23 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Light.CdXm_ABL.woff2                                    104.33 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-SemiBold.Ctx7G98q.woff2                                 105.80 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Medium.P1cOs5ep.woff2                                   105.92 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ExtraBold.B0QOs-T2.woff2                                106.11 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Bold.CuhepTt8.woff2                                     106.14 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ThinItalic.B6FKq6qK.woff2                               106.50 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Italic._3PMmu0i.woff2                                   106.88 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-BlackItalic.ZFcDzO3e.woff2                              108.75 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-LightItalic.COmGGTuG.woff2                              111.33 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ExtraLightItalic.Bf25KIHs.woff2                         111.39 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ExtraBoldItalic.OErwaGem.woff2                          111.71 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-BoldItalic.R15IsAnq.woff2                               111.81 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-SemiBoldItalic.BNz1Al2H.woff2                           112.05 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-MediumItalic.DWiIAHvp.woff2                             112.18 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Regular.DJOZHnwz.woff                                   133.84 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Thin.CVMZtQER.woff                                      135.92 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Black.DtxVgCT8.woff                                     138.76 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Light.CG9upfZ4.woff                                     140.63 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ExtraLight.CbFdmbHh.woff                                140.72 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Medium.oHczPpWt.woff                                    142.55 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ExtraBold.B8imQN9B.woff                                 142.92 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-SemiBold.BHlX_6qk.woff                                  142.93 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Bold.2PT7Q-hu.woff                                      143.21 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-Italic.Ci_5KQU1.woff                                    144.37 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ThinItalic.B2bFwwDp.woff                                145.48 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-BlackItalic.CkZ-ktDH.woff                               146.82 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ExtraLightItalic.B3gY-m5W.woff                          150.00 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-LightItalic.C2ZuFZOC.woff                               150.09 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-ExtraBoldItalic.Bt8eprh6.woff                           150.63 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-MediumItalic.8laNK8GF.woff                              150.99 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-BoldItalic.C906RMVC.woff                                151.05 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/Inter-SemiBoldItalic.CxeAX7g8.woff                            151.18 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/VennDiagram.O0wiVdv0.css                                       84.01 kB
[DEBUG]:  .svelte-kit/output/server/_app/immutable/assets/_layout.BvXVVLdn.css                                           86.10 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/settings/_layout.server.js                                              0.05 kB
[DEBUG]:  .svelte-kit/output/server/chunks/node.js                                                                        0.08 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/api/customFormattingSettings.json/GET.json/_server.js               0.09 kB
[DEBUG]:  .svelte-kit/output/server/chunks/profile.js                                                                     0.09 kB
[DEBUG]:  .svelte-kit/output/server/chunks/ssr2.js                                                                        0.10 kB
[DEBUG]:  .svelte-kit/output/server/chunks/index5.js                                                                      0.13 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/settings/_page.js                                                       0.15 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/explore/_layout.js                                                      0.15 kB
[DEBUG]:  .svelte-kit/output/server/entries/fallbacks/layout.svelte.js                                                    0.24 kB
[DEBUG]:  .svelte-kit/output/server/chunks/environment.js                                                                 0.25 kB
[DEBUG]:  .svelte-kit/output/server/chunks/paths.js                                                                       0.28 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/api/prerendered_queries/_query_hash_.arrow/_server.js               0.36 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/fix-tprotocol-service-worker.js/_server.js                          0.38 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/api/_route_hash_/_additional_hash_/all-queries.json/_server.js      0.39 kB
[DEBUG]:  .svelte-kit/output/server/internal.js                                                                           0.40 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/manifest.webmanifest/_server.js                                     0.40 kB
[DEBUG]:  .svelte-kit/output/server/chunks/globalContexts.js                                                              0.66 kB
[DEBUG]:  .svelte-kit/output/server/chunks/stores2.js                                                                     0.69 kB
[DEBUG]:  .svelte-kit/output/server/chunks/debug.js                                                                       0.71 kB
[DEBUG]:  .svelte-kit/output/server/chunks/hooks.server.js                                                                0.72 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/settings/_page.svelte.js                                                0.80 kB
[DEBUG]:  .svelte-kit/output/server/chunks/index6.js                                                                      1.22 kB
[DEBUG]:  .svelte-kit/output/server/chunks/index8.js                                                                      1.37 kB
[DEBUG]:  .svelte-kit/output/server/chunks/utils.js                                                                       1.64 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/api/pagesManifest.json/_server.js                                   2.09 kB
[DEBUG]:  .svelte-kit/output/server/chunks/index.js                                                                       2.35 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/api/_...route_/evidencemeta.json/_server.js                         2.37 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/api/settings.json/_server.js                                        2.58 kB
[DEBUG]:  .svelte-kit/output/server/chunks/index2.js                                                                      2.68 kB
[DEBUG]:  .svelte-kit/output/server/entries/endpoints/api/customFormattingSettings.json/_server.js                        2.74 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/settings/_page.server.js                                                3.34 kB
[DEBUG]:  .svelte-kit/output/server/chunks/index4.js                                                                      3.56 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/_page.md.js                                                             4.17 kB
[DEBUG]:  .svelte-kit/output/server/chunks/updater.js                                                                     4.35 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/explore/schema/_page.svelte.js                                          5.16 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/_layout.js                                                              5.62 kB
[DEBUG]:  .svelte-kit/output/server/chunks/exports.js                                                                     5.94 kB
[DEBUG]:  .svelte-kit/output/server/chunks/node2.js                                                                       7.15 kB
[DEBUG]:  .svelte-kit/output/server/chunks/ssr.js                                                                         7.65 kB
[DEBUG]:  .svelte-kit/output/server/chunks/internal.js                                                                    8.71 kB
[DEBUG]:  .svelte-kit/output/server/chunks/index7.js                                                                     13.53 kB
[DEBUG]:  .svelte-kit/output/server/chunks/inferColumnTypes.js                                                           19.82 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/_error.svelte.js                                                       25.73 kB
[DEBUG]:  .svelte-kit/output/server/chunks/stores.js                                                                     43.78 kB
[DEBUG]:  .svelte-kit/output/server/chunks/utils2.js                                                                     72.28 kB
[DEBUG]:  .svelte-kit/output/server/index.js                                                                            111.98 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/explore/console/_page.svelte.js                                       134.40 kB
[DEBUG]:  .svelte-kit/output/server/entries/pages/_layout.svelte.js                                                     217.69 kB
[DEBUG]:  .svelte-kit/output/server/chunks/VennDiagram.svelte_svelte_type_style_lang.js                               1,334.70 kB
[DEBUG]:  .svelte-kit/output/server/chunks/index3.js                                                                  3,139.85 kB
[DEBUG]:  ✓ built in 40.96s

Run npm run preview to preview your production build locally.

> Using @sveltejs/adapter-static
  Wrote site to "./build"
  ✔ done
Build complete --> ./build
```
</details>


<details>
<summary>With <code>logLevel: 'silent'</code></summary>

```
❯ pnpm build --debug

> my-evidence-project@0.0.1 build /home/zach/code/evidence/evidence/e2e/basic
> evidence build "--debug"

Evidence running with debug logging
[DEBUG]:  vite v5.4.6 building SSR bundle for production...
No sources found, execute "npm run sources" to generate
[DEBUG]:  vite v5.4.6 building for production...
[DEBUG]:  [plugin:vite:resolve] [plugin vite:resolve] Module "fs/promises" has been externalized for browser compatibility, imported by "/home/zach/code/evidence/evidence/e2e/basic/.evidence/template/src/pages/+layout.js". See https://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility for more details.
[DEBUG]:  [plugin:vite:reporter] [plugin vite:reporter]
(!) /home/zach/code/evidence/evidence/node_modules/.pnpm/prismjs@1.29.0/node_modules/prismjs/prism.js is dynamically imported by /home/zach/code/evidence/evidence/packages/ui/core-components/dist/unsorted/ui/prismLoader.js but also statically imported by /home/zach/code/evidence/evidence/packages/ui/core-components/dist/unsorted/ui/QueryViewerSupport/Prismjs.svelte, /home/zach/code/evidence/evidence/packages/ui/core-components/dist/unsorted/ui/prism-svelte.js, dynamic import will not move module into another chunk.

[DEBUG]:  ✓ built in 24.11s
No sources found, execute "npm run sources" to generate
[DEBUG]:  ✓ built in 40.82s

Run npm run preview to preview your production build locally.

> Using @sveltejs/adapter-static
  Wrote site to "./build"
  ✔ done
Build complete --> ./build
```
</details>

Reply:
Huh thought it was showing up for me but I guess I hadn't rebuilt

Considering it seems like we really just want the existing behaviour when debug mode is active, I think I'll just switch everything in this PR to be behind an `if (!isDebug())`

Reply:
@ItsMeBrianD only semi related but is there any reason we broke `svelte.config.js` into its own file but `vite.config.js` is still inlined in `build-template.js`?